test/extended/fixtures/forcepull-false-cust-bldr.json

```diff
@@ -0,0 +1,32 @@
+{
+    "kind": "BuildConfig",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "forcepull-false-cust-bldr",
+        "creationTimestamp": null,
+        "labels": {
+            "name": "forcepull-false-cust-bldr"
+        }
+    },
+    "spec": {
+        "triggers": [],
+        "source": {
+            "type": "Git",
+            "git": {
+		"uri": "https://github.com/gabemontero/cust-forcepull-false-origin-custom-docker-builder.git"
```

Comment:
I don't understand why you have separate repos + imagestreams defined for "false" and "true"?  why can't you use the same builder image for both the forcepull=false and forcepull=true test?  (question applies to all 3 build strategies).

also of course once clayton's "build from dockerfile" PR merges, we'll want to change these definitions to use that so they're not dependent on your external github repos.